acl_loader/main.py

```diff
@@ -453,6 +468,27 @@ def convert_ip(self, table_name, rule_idx, rule):
 
         return rule_props
 
+    def convert_icmp(self, table_name, rule_idx, rule):
+        rule_props = {}
+
+        is_table_v6 = self.is_table_ipv6(table_name)
+        type_key = "ICMPV6_TYPE" if is_table_v6 else "ICMP_TYPE"
+        code_key = "ICMPV6_CODE" if is_table_v6 else "ICMP_CODE"
+
+        if rule.icmp.config.type:
```

Comment:
>type [](start = 28, length = 4)

Is 0 a valid type?

Reply:
I'll add this to the test cases but "0" should work. openconfig uses 0 as a default value, so checking `== 0` will actually add ICMP_TYPE = 0 to all rules.

Reply:
Update: this was a very good catch! The sonic-acl-extension defines ICMP as a uint, so we have this default value == 0 issue. I need to patch it to treat ICMP type and code as strings so that we can use `0` as a valid code/type.